homeassistant/components/onewire/sensor.py

```diff
@@ -224,6 +239,11 @@ def unit_of_measurement(self):
         """Return the unit the value is expressed in."""
         return self._unit_of_measurement
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the sensor."""
+        return {"device_file": self._device_file, "raw_value": self._value_raw}
```

Comment:
Let's remove `device_file` from state attributes and instead use it as unique_id.

Reply:
Is this what you had in mind?


diff --git "a/homeassistant/components/onewire/sensor.py" "b/homeassistant/components/onewire/sensor.py"
index ea0c5bf66..24a64ef71 100644
--- "a/homeassistant/components/onewire/sensor.py"
+++ "b/homeassistant/components/onewire/sensor.py"
@@ -242,8 +242,12 @@ class OneWire(Entity):
     @property
     def device_state_attributes(self):
         """Return the state attributes of the sensor."""
-        return {"device_file": self._device_file, "raw_value": self._value_raw}
+        return {"device_file": self.unique_id, "raw_value": self._value_raw}
 
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return self._device_file
 
 class OneWireProxy(OneWire):
     """Implementation of a One wire Sensor through owserver."""

Reply:
Do we still need the device file in the state attributes?

Reply:
I cannot see the unique_id anywhere otherwise.... 
Among states the unique_id is not shown.

Reply:
It's not shown, but the user can rename entity_id and name, with unique_id set.

I guess it's still good to know the original device path.

Reply:
So yeah, I think that change is ok.